src/elements/Link/Link.tsx

```diff
@@ -24,7 +22,8 @@ export const Link = styled.a<LinkProps>`
     color: ${color("black100")};
   }
   &:focus {
-    border: 1px solid ${color("purple100")};
```

Comment:
Curious about the accessibility implications of this — will e.g. keyboard-only users get enough of a visual affordance when tabbing through links?

Reply:
It was very helpful to see the border as an indicator when only using the keyboard for navigating, but the border also looked more disruptive for common use cases than being accessibility-friendly. So even just a normal click was adding a border:

![clicking-link](https://user-images.githubusercontent.com/386234/49835764-3c052480-fd6e-11e8-99bd-5404769d3a96.gif)

Perhaps @nicoleyeo @briansw can comment on this?

Reply:
One thing with this border is that in case we need to keep it - we might be forced to also add a transparent border for all other states to prevent a pixel jump on focus.

Reply:
> we might be forced to also add a transparent border for all other states to prevent a pixel jump on focus.

I'm not sure if that's a reasonable change since I don't think we want the border in the first place in almost any case.

Reply:
@yuki24 - is it possible to add the border without the visual shift?